### PR TITLE
docs(root): add canary component guidance

### PR DESCRIPTION
**Pull request by GCHQ-Developer-530**
## Summary of the changes

Add canary component guidance to versioning page

## Related issue

#729 

## Checklist

- [x] I have [manually accessibility tested](https://design.sis.gov.uk/accessibility/testing/manual-testing) any changes, if relevant.
- [x] I have raised this pull request against the [develop branch](https://github.com/mi6/ic-design-system/tree/develop)

### CODE REVIEW

File: src/content/structured/get-started/releases-versioning.mdx [hunk 1]
```diff
@@ -72,6 +72,40 @@ We consider Beta components acceptable for use in production-ready applications,
 
 We regularly review the stability and reliability of Beta components. When the component meets the criteria it can be transitioned to a stable release.
 
+## Canary components
+
+Canary components are unstable components in our versioning system that are released for testing purposes.
+
+These components should not be used in production apps without understanding the risk that unexpected changes may be made to them, causing changes within the applications using them.
+
+We consider a canary component acceptable for moving to Beta once customers have tested its usage and provided enough feedback suggesting they're happy with it.
```
Comment (GCHQ-Developer-847) [inline src/content/structured/get-started/releases-versioning.mdx:81]:
```suggestion
We consider a canary component to be acceptable for moving to Beta once customers have tested its usage and provided enough feedback suggesting they're happy with it.
```

File: src/content/structured/get-started/releases-versioning.mdx [hunk 2]
```diff
@@ -72,6 +72,40 @@ We consider Beta components acceptable for use in production-ready applications,
...
+
+Canary components can be installed in the same way as normal `@ukic/` packages, but prefixed with canary e.g `@ukic/canary-react` or `@ukic/canary-web-components`. You can update to the latest canary version just as you would with normal `@ukic/` packages.
```
Comment (GCHQ-Developer-847) [inline src/content/structured/get-started/releases-versioning.mdx:83]:
```suggestion
Canary components can be installed in the same way as normal `@ukic/` packages, but prefixed with canary, e.g. `@ukic/canary-react` or `@ukic/canary-web-components`. You can update to the latest canary version just as you would with normal `@ukic/` packages.
```

File: src/content/structured/get-started/releases-versioning.mdx [hunk 3]
```diff
@@ -72,6 +72,40 @@ We consider Beta components acceptable for use in production-ready applications,
...
+  >
+    React
+  </ic-link>{" "}
+  or{" "}
+  <ic-link
+    target="_blank"
+    href="https://mi6.github.io/ic-ui-kit/branches/canary/main/web-components/?path=/docs/getting-started--docs"
+    rel="noreferer noopener nofollow"
+  >
+    Web components
```
Comment (GCHQ-Developer-847) [inline src/content/structured/get-started/releases-versioning.mdx:104]:
```suggestion
    web components
```

File: src/content/structured/get-started/releases-versioning.mdx [hunk 4]
```diff
@@ -72,6 +72,40 @@ We consider Beta components acceptable for use in production-ready applications,
...
+
+We consider a canary component acceptable for moving to Beta once customers have tested its usage and provided enough feedback suggesting they're happy with it.
+
+Canary components can be installed in the same way as normal `@ukic/` packages, but prefixed with canary e.g `@ukic/canary-react` or `@ukic/canary-web-components`. You can update to the latest canary version just as you would with normal `@ukic/` packages.
+
+A difference between canary packages and normal `@ukic/` packages is that canary package versions will follow a `<version>-canary.<build-number>` pattern.
+
+<p>
+  Another difference is that docs for canary components won't appear in the
+  components guidance section of this site. Instead, docs can be viewed in our
```
Comment (GCHQ-Developer-847) [inline src/content/structured/get-started/releases-versioning.mdx:89]: Perhaps this should be capitalised as it is referring to the actual section on the site? I.e. which is alongside Styles etc.
```suggestion
  Components guidance section of this site. Instead, docs can be viewed in our
```

**Review by GCHQ-Developer-847 — CHANGES_REQUESTED** [review]

**Review by GCHQ-Developer-299 — APPROVED** [review]

**Review by GCHQ-Developer-465 — DISMISSED** [review]

**Review by GCHQ-Developer-847 — APPROVED** [review]

File: src/content/structured/get-started/releases-versioning.mdx [hunk 5]
```diff
@@ -72,6 +72,40 @@ We consider Beta components acceptable for use in production-ready applications,
 
 We regularly review the stability and reliability of Beta components. When the component meets the criteria it can be transitioned to a stable release.
 
+## Canary components
+
+Canary components are unstable components in our versioning system that are released for testing purposes.
```
Comment (jd239) [inline src/content/structured/get-started/releases-versioning.mdx:77]:
```suggestion
Canary components are unstable components that are released for testing purposes.
```

File: src/content/structured/get-started/releases-versioning.mdx [hunk 6]
```diff
@@ -72,6 +72,40 @@ We consider Beta components acceptable for use in production-ready applications,
...
+
+These components should not be used in production apps without understanding the risk that unexpected changes may be made to them, causing changes within the applications using them.
```
Comment (jd239) [inline src/content/structured/get-started/releases-versioning.mdx:79]:
```suggestion
These components should not be used in production apps without understanding the risk that unexpected changes may be occur in order to fix bugs or improve functionality.
```

File: src/content/structured/get-started/releases-versioning.mdx [hunk 7]
```diff
@@ -72,6 +72,40 @@ We consider Beta components acceptable for use in production-ready applications,
...
+
+We consider a canary component to be acceptable for moving to Beta once customers have tested its usage and provided enough feedback suggesting they're happy with it.
+
+Canary components can be installed in the same way as normal `@ukic/` packages, but prefixed with canary, e.g. `@ukic/canary-react` or `@ukic/canary-web-components`. You can update to the latest canary version just as you would with normal `@ukic/` packages.
```
Comment (jd239) [inline src/content/structured/get-started/releases-versioning.mdx:83]:
```suggestion
Canary components can be installed in the same way as normal `@ukic/` packages, but prefixed with canary, e.g. `@ukic/canary-react` or `@ukic/canary-web-components`. 
```

Wondering if the last sentence can be removed and add the command just like it's done on the 'Get the components' pages, so something like ....

```
// Canary Web Components
npm i @ukic/canary-web-components @ukic/fonts

// Canary React
npm i @ukic/canary-react @ukic/fonts
```

**Review by jd239 — CHANGES_REQUESTED** [review]
Wondering if we change the language a bit so we specify the 'normal' @ukic packages as our core packages.

**Review by GCHQ-Developer-299 — APPROVED** [review]

File: src/content/structured/get-started/releases-versioning.mdx [hunk 8]
```diff
@@ -72,6 +72,48 @@ We consider Beta components acceptable for use in production-ready applications,
 
 We regularly review the stability and reliability of Beta components. When the component meets the criteria it can be transitioned to a stable release.
 
+## Canary components
+
+Canary components are unstable components that are released for testing purposes.
+
+These components should not be used in production apps without understanding the risk that unexpected changes may be occur in order to fix bugs or improve functionality.
```
Comment (jd239) [inline src/content/structured/get-started/releases-versioning.mdx:79]:
```suggestion
These components should not be used in production apps without understanding the risk that unexpected changes may occur in order to fix bugs or improve functionality.
```

**Pull request merged**